src/core/EdFi.DataManagementService.Core/Middleware/DisallowDuplicateReferencesMiddleware.cs

```diff
@@ -3,14 +3,17 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+using System.Text;
+using System.Text.RegularExpressions;
+using EdFi.DataManagementService.Core.External.Model;
 using EdFi.DataManagementService.Core.Model;
 using EdFi.DataManagementService.Core.Pipeline;
 using Microsoft.Extensions.Logging;
 using static EdFi.DataManagementService.Core.Response.FailureResponse;
 
 namespace EdFi.DataManagementService.Core.Middleware;
 
-internal class DisallowDuplicateReferencesMiddleware(ILogger logger) : IPipelineStep
+internal partial class DisallowDuplicateReferencesMiddleware(ILogger logger) : IPipelineStep
```

Comment:
Where are the unit tests for `DisallowDuplicateReferencesMiddleware` for these changes?